src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -511,6 +511,9 @@ namespace SimAirServingZones {
 			return;
 		}
 
+		// initialize on off fan PLR to 1 for those systems that do not set this variable
+		DataAirLoop::LoopOnOffFanPartLoadRatio = 1.0;
+
```

Comment:
@rraustad The variable LoopOnOffFanPartLoadRatio is used in the AirflowNetwork connection. Since the AirflowNetwork model allows a single AirLoop for the time being, it should not be an array. If you want to use this variable for other purposes, you may use another variable.